terraform/modules/addons/descheduler/versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1.0.0"
+  required_version = ">= 1.0"
```

Comment:
Module version constraints are generally only restricted by their lower bound of their requirements, which makes them applicable to a broader range of versions for better adoption without version conflicts. The minimum requirement is derived from the features used, where language level features will raise the Terraform required version, and the provider version will be dictated by the most recent argument addition used across the resources created (i.e. - when was the argument added/updated per the [CHANGELOG](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md), this dictates the minimum required provider version)